packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -84,9 +100,21 @@ class _MyHomePageState extends State<MyHomePage> {
         title: new Text(config.title),
       ),
       body: new Center(
+{{#withPluginHook}}
+        child: new Column(
```

Comment:
Take a look at the PR i just sent. I think the more clear separation in main.dart makes it a little more manageable.